libsplinter/src/store/memory.rs

```diff
@@ -37,6 +37,7 @@ use super::sqlite::ForeignKeyCustomizer;
 use super::StoreFactory;
 
 /// A `StoryFactory` backed by memory.
+#[derive(Clone)]
```

Comment:
I'm concerned that cloning the factory, while harmless in some circumstances, may better enable patterns which exhaust the connection pool size--when re-using an already allocated store may be a better option.

Reply:
Removed this commit